--- a/cmake/CMakeLists.txt	2019-11-27 08:24:31.000000000 -0500
+++ b/cmake/CMakeLists.txt	2021-02-01 22:27:08.332709872 -0500
@@ -9,7 +9,6 @@
     CompizPackage.cmake
     CompizBcop.cmake
     copy_file_install_user_env.cmake
-    recompile_gsettings_schemas_in_dir_user_env.cmake
     ${CMAKE_CURRENT_BINARY_DIR}/CompizDefaults.cmake
 )
 
--- a/postinst/CMakeLists.txt	2019-11-27 08:24:31.000000000 -0500
+++ b/postinst/CMakeLists.txt	2021-02-02 00:21:24.145075189 -0500
@@ -6,5 +6,4 @@
 	      PROPERTY GSETTINGS_LOCAL_COMPILE_INHIBIT_RULE
 	      FALSE)
 
-compiz_add_install_recompile_gsettings_schemas (${CMAKE_INSTALL_PREFIX}/share/glib-2.0/schemas)
 add_all_gsettings_schemas_to_local_recompilation_rule ()
--- a/cmake/CompizGSettings.cmake	2019-11-27 08:24:31.000000000 -0500
+++ b/cmake/CompizGSettings.cmake	2021-02-02 00:25:38.392150216 -0500
@@ -68,7 +68,7 @@
 
     if (PKG_CONFIG_TOOL AND
 	GLIB_COMPILE_SCHEMAS AND NOT
-	COMPIZ_DISABLE_SCHEMAS_INSTALL)
+    COMPIZ_DISABLE_GS_SCHEMAS_INSTALL)
 
 	# Install schema file
 	install (CODE "
@@ -104,7 +104,7 @@
 
     endif (PKG_CONFIG_TOOL AND
 	   GLIB_COMPILE_SCHEMAS AND NOT
-	   COMPIZ_DISABLE_SCHEMAS_INSTALL)
+       COMPIZ_DISABLE_GS_SCHEMAS_INSTALL)
 endfunction ()
 
 function (add_gsettings_local_recompilation_rule _schemas)
