--- a/compizconfig/compizconfig-python/CMakeLists.txt	2019-11-27 08:24:31.000000000 -0500
+++ b/compizconfig/compizconfig-python/CMakeLists.txt	2021-01-29 16:40:42.372316121 -0500
@@ -16,13 +16,13 @@
 set (PY_COMPIZCONFIG_SRC ${CMAKE_CURRENT_BINARY_DIR}/compizconfig.c)
 set (PY_COMPIZCONFIG_CYTHON ${CMAKE_CURRENT_SOURCE_DIR}/src/compizconfig.pyx)
 
-find_program(CYTHON_BIN cython3)
+find_program(CYTHON_BIN cython)
 if(NOT CYTHON_BIN)
   message(FATAL_ERROR "Can not build because cython binary not found.")
 endif()
 
 add_custom_command (OUTPUT ${PY_COMPIZCONFIG_SRC}
-		    COMMAND ${CYTHON_BIN} -o ${PY_COMPIZCONFIG_SRC} ${PY_COMPIZCONFIG_CYTHON}
+		    COMMAND ${CYTHON_BIN} -3 -o ${PY_COMPIZCONFIG_SRC} ${PY_COMPIZCONFIG_CYTHON}
 		    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
 		    DEPENDS ${PY_COMPIZCONFIG_CYTHON})
 
