--- a/cmake/CompizGSettings.cmake	2019-11-27 08:24:31.000000000 -0500
+++ b/cmake/CompizGSettings.cmake	2021-02-01 21:48:37.459408556 -0500
@@ -68,7 +68,7 @@
 
     if (PKG_CONFIG_TOOL AND
 	GLIB_COMPILE_SCHEMAS AND NOT
-	COMPIZ_DISABLE_SCHEMAS_INSTALL)
+    COMPIZ_DISABLE_GS_SCHEMAS_INSTALL)
 
 	# Install schema file
 	install (CODE "
@@ -104,7 +104,7 @@
 
     endif (PKG_CONFIG_TOOL AND
 	   GLIB_COMPILE_SCHEMAS AND NOT
-	   COMPIZ_DISABLE_SCHEMAS_INSTALL)
+       COMPIZ_DISABLE_GS_SCHEMAS_INSTALL)
 endfunction ()
 
 function (add_gsettings_local_recompilation_rule _schemas)
--- a/cmake/CMakeLists.txt	2019-11-27 08:24:31.000000000 -0500
+++ b/cmake/CMakeLists.txt	2021-02-01 22:27:08.332709872 -0500
@@ -9,7 +9,6 @@
     CompizPackage.cmake
     CompizBcop.cmake
     copy_file_install_user_env.cmake
-    recompile_gsettings_schemas_in_dir_user_env.cmake
     ${CMAKE_CURRENT_BINARY_DIR}/CompizDefaults.cmake
 )
 
