--- a/src/main.c	2011-04-21 08:06:20.000000000 -0400
+++ b/src/main.c	2021-02-04 18:31:56.000000000 -0500
@@ -504,8 +504,9 @@
     else
 	maxextents = extents;
 
-    decor_quads_to_property(data, GDK_PIXMAP_XID(d->pixmap),
-			    &extents, &maxextents, 0, 0, quads, nQuad);
+    decor_quads_to_property(data, 0, GDK_PIXMAP_XID(d->pixmap),
+			    &extents, &extents, &maxextents, &maxextents,
+                0, 0, quads, nQuad, 0xffffff, 0, 0);
 
     gdk_error_trap_push();
     XChangeProperty(xdisplay, d->prop_xid,
@@ -1936,8 +1937,9 @@
 
     nQuad = set_switcher_quads(quads, d->width, d->height, ws);
 
-    decor_quads_to_property(data, GDK_PIXMAP_XID(d->pixmap),
-			    &extents, &extents, 0, 0, quads, nQuad);
+    decor_quads_to_property(data, 0, GDK_PIXMAP_XID(d->pixmap),
+			    &extents, &extents, &extents, &extents,
+                0, 0, quads, nQuad, 0xffffff, 0, 0);
 
     style = gtk_widget_get_style (style_window);
 
@@ -2352,7 +2354,7 @@
     xroot = RootWindowOfScreen(gdk_x11_screen_get_xscreen(screen));
 
     bareAtom = XInternAtom(xdisplay, DECOR_BARE_ATOM_NAME, FALSE);
-    normalAtom = XInternAtom(xdisplay, DECOR_NORMAL_ATOM_NAME, FALSE);
+    normalAtom = XInternAtom(xdisplay, DECOR_WINDOW_ATOM_NAME, FALSE);
     activeAtom = XInternAtom(xdisplay, DECOR_ACTIVE_ATOM_NAME, FALSE);
 
     if (ws->shadow_pixmap)
@@ -2363,9 +2365,9 @@
 
 	nQuad = set_shadow_quads(quads, width, height, ws);
 
-	decor_quads_to_property(data, GDK_PIXMAP_XID(ws->shadow_pixmap),
-				&ws->shadow_extents, &ws->shadow_extents, 0, 0,
-				quads, nQuad);
+	decor_quads_to_property(data, 0, GDK_PIXMAP_XID(ws->shadow_pixmap),
+				&ws->shadow_extents, &ws->shadow_extents, &ws->shadow_extents, &ws->shadow_extents,
+                0, 0, quads, nQuad, 0xffffff, 0, 0);
 
 	XChangeProperty(xdisplay, xroot,
 			bareAtom,
@@ -2423,8 +2425,9 @@
 
 	(*d.draw) (&d);
 
-	decor_quads_to_property(data, GDK_PIXMAP_XID(d.p_inactive),
-				&extents, &extents, 0, 0, quads, nQuad);
+	decor_quads_to_property(data, 0, GDK_PIXMAP_XID(d.p_inactive),
+				&extents, &extents, &extents, &extents, 
+                0, 0, quads, nQuad, 0xffffff, 0, 0);
 
 	XChangeProperty(xdisplay, xroot,
 			normalAtom,
@@ -2432,14 +2435,15 @@
 			32, PropModeReplace, (guchar *) data,
 			BASE_PROP_SIZE + QUAD_PROP_SIZE * nQuad);
 
-	decor_quads_to_property(data, GDK_PIXMAP_XID(d.p_active),
-				&extents, &extents, 0, 0, quads, nQuad);
+	decor_quads_to_property(data, 0, GDK_PIXMAP_XID(d.p_active),
+				&extents, &extents, &extents, &extents,
+                0, 0, quads, nQuad, 0xffffff, 0, 0);
 
 	XChangeProperty(xdisplay, xroot,
 			activeAtom,
 			XA_INTEGER,
 			32, PropModeReplace, (guchar *) data,
-			BASE_PROP_SIZE + QUAD_PROP_SIZE * nQuad);
+			PROP_HEADER_SIZE + BASE_PROP_SIZE + QUAD_PROP_SIZE * N_QUADS_MAX);
     }
 }
 
@@ -5617,7 +5621,7 @@
     update_window_extents(ws);
     update_shadow(pfs);
 
-    decor_set_dm_check_hint(xdisplay, DefaultScreen(xdisplay));
+    decor_set_dm_check_hint(xdisplay, DefaultScreen(xdisplay), WINDOW_DECORATION_TYPE_PIXMAP);
 
     update_settings(ws);
 
